torch/csrc/dynamo/guards.cpp

```diff
@@ -782,12 +784,14 @@ class DictGuardManager;
 class LeafGuard {
  public:
   // Most guards do not need root guard manager.
-  LeafGuard(py::object verbose_code_parts)
+  LeafGuard(const py::object& verbose_code_parts)
```

Comment:
Shouldn't these be moved by value or am I missing something?

Reply:
Because py::object is converted into py::list and the conversion takes reference.